src/main/java/in/org/projecteka/hiu/consent/model/consentmanager/Frequency.java

```diff
@@ -0,0 +1,17 @@
+package in.org.projecteka.hiu.consent.model.consentmanager;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
+@Data
+public class Frequency {
+    private Unit unit;
+    private int value;
```

Comment:
Additional: You can have static fields like ZERO_HOUR => new Frequency(Unit.Hour, 0) and use it in the to method.